src/renderer/template/misc.css

```diff
@@ -192,6 +192,7 @@
   border-left: 1px solid #d8d8d8;
   flex: 4 4 327px;
   min-width: 327px;
+  overflow: hidden;
```

Comment:
Why did you add this?

Reply:
Without it a second scrollbar appears at the right of the application when the statusbar is enabled. Not the editor's, but one for the entire mainbar region.